python/paddle/fluid/contrib/slim/tests/test_mkldnn_int8_quantization_strategy.py

```diff
@@ -0,0 +1,134 @@
+
+
+class TestMKLDNNPostTrainingQuantStrategy(TestCalibration):
+    """
+    Test API of Post Training quantization strategy for int8 with MKLDNN.
+    """
+
+    def _reader_creator(self, data_file='data.bin', cycle=False):
+        def reader():
+            with open(data_file, 'rb') as fp:
+                num = fp.read(8)
+                num = struct.unpack('q', num)[0]
+                imgs_offset = 8
+                img_ch = 3
+                img_w = 224
+                img_h = 224
+                img_pixel_size = 4
+                img_size = img_ch * img_h * img_w * img_pixel_size
+                label_size = 8
+                labels_offset = imgs_offset + num * img_size
+                step = 0
+
+                while step < num:
+                    fp.seek(imgs_offset + img_size * step)
+                    img = fp.read(img_size)
+                    img = struct.unpack_from('{}f'.format(img_ch * img_w *
+                                                          img_h), img)
+                    img = np.array(img)
+                    img.shape = (img_ch, img_w, img_h)
+                    fp.seek(labels_offset + label_size * step)
+                    label = fp.read(label_size)
+                    label = struct.unpack('q', label)[0]
+                    yield img, int(label)
+                    step += 1
+                    if cycle and step == num:
+                        step = 0
+
+        return reader
+
+    def _update_config_file(self, model_name):
+        config_path = './quantization/config_mkldnn_int8.yaml'
+        new_config_path = './quantization/{0}.yaml'.format(model_name)
+        shutil.copy(config_path, new_config_path)
+
+        with open(new_config_path, 'r+') as fp:
+            data = fp.read()
+        model_path = '{0}/model'.format(model_name)
+        model_path = os.path.join(self.cache_folder, model_path)
+        data = data.replace('MODEL_PATH', model_path)
+        output_path = './{0}/int8'.format(model_name)
+        data = data.replace('OUTPUT_PATH', output_path)
+        with open(new_config_path, 'w') as fp:
+            fp.write(data)
+
+        return new_config_path
+
+    def _test_int8_quant(self, data_path, config_path):
+        #warmup dataset, only use the first batch data
+        test_reader = paddle.batch(
+            self._reader_creator(data_path, False), batch_size=100)
+        com_pass = Compressor(
+            place=None,
+            scope=None,
+            train_program=None,
+            train_reader=None,
+            train_feed_list=[],
+            train_fetch_list=[],
+            eval_program=None,
+            eval_reader=test_reader,
+            eval_feed_list=[],
+            eval_fetch_list=[],
+            teacher_programs=[],
+            checkpoint_path='',
+            train_optimizer=None,
+            distiller_optimizer=None)
+        com_pass.config(config_path)
+        eval_graph = com_pass.run()

```

Comment:
Please add the accuracy result after the quantization. Otherwise, we cannot verify the quantization was done properly. I know the INT8v2 quantization is verified by the c-api unit tests, but the check should be added here as well, in my opinion.